src/NumberTheory/NmbThy.jl

```diff
@@ -154,6 +153,37 @@ function irreducibles(S::Vector{NfAbsOrdIdl{AnticNumberField,nf_elem}})
   return res
 end
 
+#= From Lars:
```

Comment:
@fieker I am confused by this PR. What is this new comment about?!

Reply:
From the context it seems to show how to use different backends for the `solve_non_negative` method.

Reply:
Indeed, It did not solve the original problem, but I thought this helpful enough so it should be preserved in a useful spot.
The original provlem involved a largish solve problem that allowed for "trivial" preprocessing, there were quite some inqualtities if the type -2*x_i > 0 or 3x <= 2 which in this context can easily be removed (the entire variable).
This was an attempt, by Lars, to try the different backends. The result was not quite as predicted by Michael...